src/lib/y2partitioner/widgets/encrypt.rb

```diff
@@ -191,6 +191,38 @@ def help_for_pervasive_luks2
         )
       end
 
+      # Help text for the Regular Luks2 encryption method
+      #
+      # @return [String]
+      def help_for_luks2
+        encrypt_method = Y2Storage::EncryptionMethod.find(:luks2)
+
+        format(
+          # TRANSLATORS: help text for Regular Luks2 encryption method
+          _("<p><b>%{label}</b>: allows to encrypt the device using LUKS2. A variant of LUKS " \
+            "(Linux Unified Key Setup) that uses a newer version of the header format. That " \
+            "allows further possibilities like setting a label to reference the LUKS device " \
+            "(for example, in the crypttab file). You have to provide the encryption password " \
+            "and the password-based key derivation function (PBKDF) that will be used to protect " \
+            "that passphrase.</p>" \
+            "<p>The function to use depends on the context, the hardware capabilities and the " \
+            "needed level of compatibility with other system components (see below).<p>" \
+            "<p><b>PBKDF2</b> refers to the function of that name, according to RFC2898. Is the " \
+            "function that LUKS1 uses.</p>" \
```

Comment:
Question: LUKS1? Sorry for my ignorance here :)

Reply:
Yes, LUKS1 always uses PBKDF2 as pbkdf, while LUKS2 can use different pbkdfs. Clear, isn't it? :wink:

Now seriously, I expected that to be clear by the context. I can try to reword it if it's not the case.

Reply:
>  Clear, isn't it?

Sure :dizzy:  :dizzy_face: :dizzy: